lib/cocoapods/installer/analyzer/target_inspector.rb

```diff
@@ -81,11 +82,6 @@ def compute_project_path
           path
         end
 
-        # @return [Xcodeproj::Project] the user's Xcode project, used for target
-        #         inspection
-        #
-        attr_accessor :user_project
```

Comment:
this didnt even need to be a property this object holds on to. It was only used within `compute_results`

Reply:
It probably dates back to when a target definition could reference multiple targets in a project